build_debian.sh

```diff
@@ -347,7 +349,9 @@ sudo LANG=C chroot $FILESYSTEM_ROOT bash -c "find /usr/share/i18n/locales/ ! -na
 # Install certain fundamental packages from $IMAGE_DISTRO-backports in order to get
 # more up-to-date (but potentially less stable) versions
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y -t $IMAGE_DISTRO-backports install \
-    picocom
+    picocom \
+    systemd \
+    systemd-sysv
```

Comment:
@stepanblyschak why do we need both: systemd & systemd-sysv?

Reply:
You are rgiht I need systemd-sysv witch brings systemd in as a dependency